test/rekt/features/broker/control_plane.go

```diff
@@ -100,7 +100,9 @@ func ControlPlaneBroker(brokerName string, brokerOpts ...manifest.CfgFn) *featur
 		Should("The class of a Broker object SHOULD be immutable.",
 			brokerClassIsImmutable).
 		Should("Set the Broker status.deadLetterSinkURI if there is a valid spec.delivery.deadLetterSink defined",
-			BrokerStatusDLSURISet)
+			BrokerStatusDLSURISet).
+		Should("Broker config SHOULD be immutable.",
+			brokerConfigIsImmutable)
```

Comment:
The spec says this is a MUST https://github.com/knative/specs/blob/8e560194c5ca5c290fa9e0003592afd2c423d57e/specs/eventing/control-plane.md?plain=1#L144

Reply:
Updated